src/qibojit/backends/gpu.py

```diff
@@ -473,7 +473,14 @@ def calculate_expectation_density_matrix(self, matrix, state, normalize):
             ev = ev / norm
         return ev
 
-    def calculate_eigenvalues(self, matrix, k=6):
+    def calculate_eigenvalues(self, matrix, k: int = 6, hermitian: bool = True):
+        if not hermitian:
+            raise_error(
+                NotImplementedError,
+                "GPU backends do not support `np.linalg.eig` "
+                + "or `np.linalg.eigvals` for non-Hermitian matrices.",
+            )
```

Comment:
```suggestion
            log.warning(
                "Falling back to CPU for eigenvalues calculation of a non-Hermitian operator."
            )

            matrix_cpu = self.to_numpy(matrix)
            eigenvalues = super().calculate_eigenvalues(
                matrix_cpu, k, hermitian
            )
            return self.cast(eigenvalues, dtype=eigenvalues.dtype)
```
I believe you forgot to update this one as well for the CPU fallback